### ISSUE TITLE
Does not work in monorepos

### ISSUE THREAD

**Issue by NullVoxPopuli** [ember-template-lint/ember-template-lint-plugin-prettier#202]
Using ember-template-lint v4
this plugin: v4

```
❯ yarn lint:hbs
yarn run v1.22.17
$ ember-template-lint .
file://<repo>/node_modules/ember-template-lint/lib/get-config.js:265
        throw new Error(`Cannot find configuration for extends: ${extendName}`);
              ^

Error: Cannot find configuration for extends: ember-template-lint-plugin-prettier
    at processExtends (file://<repo>/node_modules/ember-template-lint/lib/get-config.js:265:15)
    at getProjectConfig (file://<repo>/node_modules/ember-template-lint/lib/get-config.js:399:5)
    at async Linter.loadConfig (file://<repo>/node_modules/ember-template-lint/lib/linter.js:48:19)
    at async Linter.verify (file://<repo>/node_modules/ember-template-lint/lib/linter.js:304:7)
    at async run (file://<repo>/node_modules/ember-template-lint/bin/ember-template-lint.js:211:21)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.}
```

This is fixed by: https://github.com/ember-template-lint/ember-template-lint/pull/2359
But blocked by: https://github.com/ember-template-lint/ember-template-lint/pull/2360

cc @bmish

**Comment by runspired** [comment]
Also monorepo, also not working,, but it doesn't throw the above error it simply can't find the prettier config regardless of whether I put the config in the project root or the individual workspace.

**Comment by runspired** [comment]
I tried moving the entirety of my linting setup into root and executing it from there, no dice. Once workspaces are in play it seems that the prettier config cannot be found by ember-template-lint-plugin-prettier

**Comment by dcyriller** [comment]
Thanks for the details.

I have looked at the code. Odds are high that this plugin will work once monorepos are supported by ember-template-lint itself. Indeed, it resolves the config with a Prettier method: https://prettier.io/docs/en/api.html#prettierresolveconfigfilepath--options

**Comment by bmish** [comment]
Is this still broken? Haven't heard about the ember-template-lint monorepo issue in a while.
